frontend/app/components/encryptable/show.js

```diff
@@ -9,12 +9,16 @@ export default class ShowComponent extends Component {
   @service intl;
   @service notify;
```

Comment:
create own ember route for encryptable logs and fetch entries there.
/encryptables/42/logs
try to extract log content to own component

Reply:
Sorry, I made this Pull-Request to the master branch but it should be to the 564-LOG-encryptable-read-access branch. I changed this now. This file and the comment to it have nothing to do with our implementation of the personal logs.